INSTALLER.md

```diff
@@ -91,8 +98,157 @@ For code signing to work, you will need to set the `HF_PFX_FILE` and `HF_PFX_PAS
 1.  Perform a Release build of `package`
       Installer is now available in `build/_CPack_Packages/Darwin/DragNDrop
       
-### FAQ
-
-1. **Problem:** Failure to open a file. ```File: failed opening file "\FOLDERSHARE\XYZSRelease\...\Credits.rtf" Error in script "C:\TFS\XYZProject\Releases\NullsoftInstaller\XYZWin7Installer.nsi" on line 77 -- aborting creation process```
-    1. **Cause:** The complete path (current directory + relative path) has to be < 260 characters to any of the relevant files.
-    1. **Solution:** Move your build and packaging folder as high up in the drive as possible to prevent an overage.
+### Linux
+
+#### Server
+
+##### Ubuntu 18.04 | .deb
+
+1. Ensure you are using an Ubuntu 18.04 system, here are the recommended specs:
+    ```
+    AWS EC2 Instance Type: C5a.4xlarge
```

Comment:
Building an Ubuntu 18.04 server package only needs an amd64/x86_64 machine with at least 40gb of disk space. There is no point in recommending an AWS instance, as it is more expensive than using a virtual machine and takes much longer for a non-AWS user than pretty much anything else. Personally I build in VirtualBox with 6 threads. You can easily build the server on even just one thread.